v3/plugins/cdr/code-server/latest.txt

```diff
@@ -0,0 +1 @@
+che-v2
```

Comment:
I don't think its wise putting this on the rolling version since we want a stable plugin as much as possible.

Reply:
Yeah I went to bed on Friday thinking that exact thing :)

I'll update this when I rebase, there's a few other plugins being added now so I want to do it only once.

@sr229 What do you think about naming the `che-v2` version `next`? That's kind of more in line with what we do with e.g. Theia currently, so it may be more understandable to a user. (Basically everything would be the same except the version that shows up in the Che dashboard).

Reply:
Works for me 👍

Reply:
Updated `che-v2` version to `next` and pointed `latest` at `2.1523-vsc1.38.1`